python/bindings/src/node_attributes.cpp

```diff
@@ -150,6 +150,23 @@ void addBindings(pybind11::module_& module) {
             attrs.world_R_object = rot;
           });
 
+  py::class_<KhronosObjectAttributes, ObjectNodeAttributes>(module,
+                                                            "KhronosObjectAttributes")
+      .def(py::init<>())
+      .def_readwrite("first_observed_ns", &KhronosObjectAttributes::first_observed_ns)
+      .def_readwrite("last_observed_ns", &KhronosObjectAttributes::last_observed_ns)
+      .def(
+          "mesh",
+          [](const KhronosObjectAttributes& attrs) { return &attrs.mesh; },
+          py::return_value_policy::reference_internal)
+      .def_readwrite("trajectory_timestamps",
+                     &KhronosObjectAttributes::trajectory_timestamps)
+      .def_readwrite("trajectory_positions",
+                     &KhronosObjectAttributes::trajectory_positions)
+      .def_readwrite("dynamic_object_points",
+                     &KhronosObjectAttributes::dynamic_object_points)
```

Comment:
For future reference: these might be better as read-only (or opaque bindings). Otherwise there can be some weirdness with setting them on the python side and not seeing them on the c++ side (see [here](https://pybind11.readthedocs.io/en/stable/advanced/cast/stl.html#making-opaque-types))

Reply:
Thanks for the pointer! Making these readonly for now, can reconsider in the future if we needed write access for some reason.